.php_cs.dist

```diff
@@ -45,6 +45,7 @@ $config = PhpCsFixer\Config::create()
         'phpdoc_types_order' => true,
         'semicolon_after_instruction' => true,
         'single_line_comment_style' => true,
+        'space_after_semicolon' => true,
```

Comment:
we could follow `@Symfony` ruleset here, no need for exception

Reply:
Updated.